target/linux/ramips/dts/mt7620a_wavlink_wn530hg4.dts

```diff
@@ -0,0 +1,137 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "wavlink,wn530hg4", "ralink,mt7620a-soc";
+	model = "wavlink WN530HG4";
```

Comment:
This should be "Wavlink WL-WN530HG4" or "WAVLINK WL-WN530HG4" ...

Compatible/image/dts name should be changed accordingly: wavlink,wl-wn530hg4 etc.